lib/screens/questionaire_screen/widgets/last_cover.dart

```diff
@@ -0,0 +1,142 @@
+  final String image;
+
+  /// Function for getting a page back.
+  final void Function()? backFunction;
+
+  /// Function for getting the next page.
+  final void Function()? nextFunction;
+
+  @override
+  Widget build(BuildContext context) {
+    double? height = MediaQuery.of(context).size.height -
+        AppBar().preferredSize.height -
+        kToolbarHeight;
+    double? width = MediaQuery.of(context).size.width;
+    double? fontSize = 30;
+
+    String tips = AppLocalizations.of(context)!.tips;
+    String drinkEnough = AppLocalizations.of(context)!.drinkEnough;
+
+    GlobalTheme theme = Theme.of(context).extension<GlobalTheme>()!;
+
+    return QuestionairePage(
+      nextFunction: nextFunction,
+      backFunction: backFunction,
+      color: theme.darkGreen1!,
+      child: SizedBox(
+        width: width,
+        height: height - (45 + 15 + 10),
+        child: Column(
+          children: <Widget>[
+            SizedBox(
+              height: height / 8,
+            ),
+            PicosSvgIcon(
+              assetName: image,
+              height: 200,
+              width: 175,
+              color: Colors.white,
+            ),
+            Padding(
+              padding: const EdgeInsets.only(top: 60),
+              child: Text(
+                title,
+                textAlign: TextAlign.center,
+                style: TextStyle(
+                  fontSize: fontSize,
+                  fontWeight: FontWeight.bold,
+                  color: Colors.white,
+                ),
+              ),
+            ),
+            SizedBox(
+              height: height / 10,
+            ),
+            Container(
+              decoration: BoxDecoration(
+                border: Border.all(color: Colors.white),
+                borderRadius: const BorderRadius.all(
+                  Radius.circular(10.0),
+                ),
+              ),
+              child: Stack(
+                clipBehavior: Clip.none,
+                children: <Widget>[
+                  Padding(
+                    padding: const EdgeInsets.all(10),
+                    child: RichText(
+                      text: TextSpan(
+                        style: const TextStyle(
+                          color: Colors.white,
+                          height: 2,
+                        ),
+                        children: <TextSpan>[
+                          TextSpan(
+                            text: '$tips\n',
+                            style: const TextStyle(fontWeight: FontWeight.bold),
+                          ),
+                          TextSpan(
+                            text: drinkEnough,
+                          ),
+                        ],
+                      ),
+                    ),
+                  ),
+                  const Positioned(
+                    top: -20,
+                    right: 20,
+                    child: PicosSvgIcon(
+                      assetName: 'assets/Tipp.svg',
```

Comment:
The colors aren't rendered for this SVG.
Check this out: [https://stackoverflow.com/a/65033951](https://stackoverflow.com/a/65033951)
I've tried it. After preparing the SVG with this tool and removing the colorFilter from PicosSvgIcon, the Picture gets rendered correctly.
I would suggest to try this solution for all SVG's and watch what happens.